<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftHandUI</class>
 <widget class="QWidget" name="LeftHandUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeftHandUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LeftHandUI {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.994, y2:1, stop:0 rgba(0, 143, 143, 255), stop:0.289773 rgba(170, 170, 255, 255), stop:1 rgba(90, 90, 90, 255));
}</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxMoveModes">
     <property name="styleSheet">
      <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-radius: 15px;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 8000px 5px 8000px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="title">
      <string>Move Modes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonWander">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	font: 10pt &quot;Times New Roman&quot;;
	background-color: rgb(170, 172, 208);
}
QPushButton:hover{
	background: transparent;
	border: 2px solid #aaacd0;
}</string>
        </property>
        <property name="text">
         <string>Wander</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonFocus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	font: 10pt &quot;Times New Roman&quot;;
	background-color: rgb(170, 172, 208);
}
QPushButton:hover{
	background: transparent;
	border: 2px solid #aaacd0;
}</string>
        </property>
        <property name="text">
         <string>Focus</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMeshAlpha">
     <property name="styleSheet">
      <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-radius: 15px;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 8000px 5px 8000px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="title">
      <string>Mesh Alpha in Mix-Rendering</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelMeshAlpha">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;
padding: 5px 5px;</string>
        </property>
        <property name="text">
         <string>0.50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSliderMeshAlpha">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;Times New Roman&quot;;
padding: 5px 5px;</string>
     </property>
     <property name="text">
      <string>Set it ZERO to use real Mix-Rendering.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTF">
     <property name="styleSheet">
      <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-color: #FF17365D;
    margin-top: 27px;
    font-size: 14px;
    border-radius: 15px;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 8000px 5px 8000px;
    background-color: #FF17365D;
    color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="title">
      <string>Transfer Function</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
